rosdep/python.yaml

```diff
@@ -19,6 +19,16 @@ adafruit-mcp3008-pip:
   ubuntu:
     pip:
       packages: [Adafruit-MCP3008]
+adafruit-circuitpython-motor-pip:
```

Comment:
Since these are pip only keys, please prefix them with `python3-`. That is `adafruit-circuitpython-motor-pip` should become `python3-adafruit-circuitpython-motor-pip`.

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1